controllers/argorollouts_controller.go

```diff
@@ -47,6 +47,10 @@ type RolloutManagerReconciler struct {
 	client.Client
 	Scheme                       *runtime.Scheme
 	OpenShiftRoutePluginLocation string
+	//ArgoRolloutsCustumLabel is used to set custom labels on Argo Rollouts resources
+	// This is used to set the label on the ConfigMap and Secrets created by the controller for now.
+	// The label is used to identify the resources created by the RolloutManager.
+	ArgoRolloutsCustomLabel map[string]string
```

Comment:
```suggestion
	// ResourceLabels is used to set custom labels on Argo Rollouts resources
	// This is used to set the label on the ConfigMap and Secrets created by the controller for now.
	// The label is used to identify the resources created by the RolloutManager.
	ResourceLabels map[string]string
```

Can we rename it to `ResourceLabels` ?

Reply:
Hi siddhesh, renamed it